src/Microsoft.AspNet.Mvc.Core/DefaultActionModelConvention.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc.ApplicationModels;
+
+namespace Microsoft.AspNet.Mvc
```

Comment:
make this and the the DefaultControllerModelConvention private classes (and name them appropriately)